src/test/java/nextstep/subway/path/acceptance/pathAcceptanceTest.java

```diff
@@ -0,0 +1,155 @@
+    @DisplayName("경로 조회를 한다.")
+    @Test
+    public void paths_find_success() {
+        // when
+        var 경로_조회_결과 = RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().get("/paths?source=" + 교대역.getId() + "&target=" + 양재역.getId())
+                .then().log().all()
+                .extract().response().body().as(PathResponse.class);
+
+        // then
+        assertEquals(경로_조회_결과.getStationResponseList(), List.of(교대역, 남부터미널역, 양재역));
+        assertEquals(경로_조회_결과.getDistance(), 5L);
+    }
+
+    /* Given: 지하철 역과 지하철 노선이 등록되어 있고,
+       When : 관리자가 출발역의 ID와 도착역의 ID가 동일한 paths를 요청하면,
+       Then : 관리자는 경로의 구간 조회에 실패한다. */
+    @DisplayName("경로 조회를 실패한다. 출발역과 도착역은 동일할 수 없다.")
+    @Test
+    public void paths_find_fail() {
+        // when
+        var 경로_조회_결과 = RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().get("/paths?source=" + 교대역.getId() + "&target=" + 교대역.getId())
+                .then().log().all()
+                .extract().as(ErrorResponse.class);
+
+        // then
+        Assertions.assertAll(
+                () -> assertEquals(경로_조회_결과.getStatus(), PATH_DUPLICATE_STATION.getStatus()),
+                () -> assertEquals(경로_조회_결과.getCode(), PATH_DUPLICATE_STATION.getCode()),
+                () -> assertEquals(경로_조회_결과.getDescription(), PATH_DUPLICATE_STATION.getDescription())
+        );
+
+    }
+
+    /* Given: 지하철 역과 지하철 노선이 등록되어 있고,
+       When : 관리자가 연결되어 있지 않은 출발역의 ID와 도착역의 ID를 통해 paths를 요청하면,
+       Then : 관리자는 경로의 구간 조회에 실패한다. */
+    @DisplayName("경로 조회를 실패한다. 출발역과 도착역은 연결되어 있어야한다.")
+    @Test
+    public void paths_find_fail2() {
+        // when
+        var 경로_조회_결과 = RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().get("/paths?source=" + 교대역.getId() + "&target=" + 용산역.getId())
+                .then().log().all()
+                .extract().as(ErrorResponse.class);
+
+        // then
+        Assertions.assertAll(
+                () -> assertEquals(경로_조회_결과.getStatus(), PATH_NOT_FOUND.getStatus()),
+                () -> assertEquals(경로_조회_결과.getCode(), PATH_NOT_FOUND.getCode()),
+                () -> assertEquals(경로_조회_결과.getDescription(), PATH_NOT_FOUND.getDescription())
+        );
+
+    }
+
+    /* Given: 지하철 역과 지하철 노선이 등록되어 있고,
+       When : 관리자가 존재하지 않은 출발역의 ID와 도착역의 ID를 통해 paths를 요청하면,
+       Then : 관리자는 경로의 구간 조회에 실패한다. */
+    @DisplayName("경로 조회를 실패한다. 존재하는 출발역과 도착역만 경로 조회가 가능하다.")
+    @Test
+    public void paths_find_fail3() {
+        // when
+        var 경로_조회_결과 = RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().get("/paths?source=" + "10" + "&target=" + "11")
+                .then().log().all()
+                .extract().as(ErrorResponse.class);
```

Comment:
```suggestion
        // given
        var 존재하지_않는_역_ID_1 = "10"
        var 존재하지_않는_역_ID_2 = "10"
        // when
        var 경로_조회_결과 = RestAssured.given().log().all()
                .contentType(MediaType.APPLICATION_JSON_VALUE)
                .when().get("/paths?source=" + 존재하지_않는_역_ID_1  + "&target=" + 존재하지_않는_역_ID_2)
                .then().log().all()
                .extract().as(ErrorResponse.class);
```

10과 11을 쓰는 것보다, 의미 있는 이름으로 선언하는 게 좋겠죠?